### ISSUE TITLE
Disable the Ignore update button.

### ISSUE THREAD

**Issue by Shaewen-Chronicles** [winsontan520/Android-WVersionManager#6]
Is ther the possibility to disable this button?

**Comment by rohitkhajuria** [comment]
Did you get the solution? am also facing the same problem

**Comment by GOPALAKRISHNAREDDY** [comment]
Yes, I have already posted the solution in the GitHub, so u can refer there.

From
K.GOPALA KRISHNA REDDY
(9951103566)
Sent from Mail for Windows 10
